src/main/java/org/fogbowcloud/saps/engine/core/catalog/jdbc/JDBCCatalogUtil.java

```diff
@@ -0,0 +1,47 @@
+package org.fogbowcloud.saps.engine.core.catalog.jdbc;
+
+import org.fogbowcloud.saps.engine.core.catalog.exceptions.CatalogException;
+import org.fogbowcloud.saps.engine.core.model.SapsImage;
+import org.fogbowcloud.saps.engine.core.model.SapsUser;
+import org.fogbowcloud.saps.engine.core.model.enums.ImageTaskState;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JDBCCatalogUtil {
+
```

Comment:
Exceptions of type JDBCCatalogException should be thrown instead of CatalogException for this whole class.

Reply:
Done